UILabel_Typography_Extensions/Typography/UILabel+Typograpy.swift

```diff
@@ -67,20 +73,39 @@ extension UILabel: TypographyExtensions {
 	}
 }
 
-
 extension UILabel {
-	
+
+    // MARK: Stored properties
+
+    private enum Keys {
+        static var attributesForEmptyString: UInt8 = 0
+    }
+
+    private var attributesForEmptyString: [NSAttributedString.Key: Any]? {
+        get {
+            objc_getAssociatedObject(self, &Keys.attributesForEmptyString) as? [NSAttributedString.Key: Any]
+        }
+        set {
+            objc_setAssociatedObject(self, &Keys.attributesForEmptyString, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+        }
+    }
```

Comment:
I'm thinking of a more simplified solution for this, simply storing an `NSAttributedString` that holds the attributes even when the label is empty, not sure.